airbyte-bootloader/src/main/java/io/airbyte/bootloader/BootloaderApp.java

```diff
@@ -249,7 +291,11 @@ public static void main(final String[] args) throws Exception {
       // Ensure that the database resources are closed on application shutdown
       CloseableShutdownHook.registerRuntimeShutdownHook(configsDataSource, jobsDataSource, configsDslContext, jobsDslContext);
 
-      final var bootloader = new BootloaderApp(configs, featureFlags, secretMigrator, configsDslContext, jobsDslContext, configsFlyway, jobsFlyway);
+      final DefinitionsProvider definitionsProvider = getLocalDefinitionsProvider();
+
+      final var bootloader =
+          new BootloaderApp(configs, featureFlags, secretMigrator, configsDslContext, jobsDslContext, configsFlyway, jobsFlyway, definitionsProvider,
+              false);
```

Comment:
I assume that this is hard-coded to `false` just for now and will eventually be replaced by some sort of configuration property?

Reply:
Yes, I had the change to add the configs, however it felt more misleading for the time being because this how we apply upgrades is currently hardcoded and injected in cloud.
The config should control both how apply the upgrades and how we validate.
Unless someone has a strong feeling we should address this now, I'd rather push it to when we micronaut the bootloader.

Should I move [this comment](https://github.com/airbytehq/airbyte/pull/19200/files#diff-887184626b5be3420f725a31a95dfde933c7e85f678d7f8de66ea6caf86462d4R82) to here instead?